### ISSUE TITLE
libc++1 kept back after 20.04 -> 20.10 upgrade with Discord installed

### ISSUE THREAD

**Issue by jacobgkau** [pop-os/upgrade#163]
Steps to recreate:
- Perform clean installation of Pop!_OS 20.04 LTS.
- Install all available updates via the Pop!_Shop.
- Install the Discord .deb package via the Pop!_Shop.
- Upgrade to 20.10.

Actual behavior:
- Upon logging in after the upgrade, I get a notification from the Pop!_Shop that updates are available.
- Running a `sudo apt update` shows `1 package can be upgraded`.
- Running a `sudo apt upgrade` shows `The following packages have been kept back: libc++1`.

This is occurring on the `master` and `rec` branches.

**Comment by jacobgkau** [comment]
This is still an issue. A `full-upgrade` does not work around the issue, but running `apt policy libc++1` and then forcing the installation of the candidate version with `sudo apt install libc++1=<version number>` does work around the issue.

**Comment by jacobgkau** [comment]
I tried adding `libc++1` to the preinstall list in `/usr/lib/pop-upgrade/upgrade.sh`, and it did not make a difference.

**Comment by mmstick** [comment]
What if we have this in the install script at the beginning?

```sh
if dpkg -s libc++1 &>/dev/null; then
    apt-get install libc++1=$(apt-cache policy libc++1 | grep Candidate | awk '{print $2}')
fi
```

To install the candidate version when the package is installed.

**Comment by jacobgkau** [comment]
I tried adding that statement in two places. First, I put it right underneath the `PREINST` definition at the top of the script; it had no effect there. The second time, I put it at the top of the `apt_full_upgrade` function. This second time, I ran the upgrade script manually so I could observe whether the `if` statement was running, and I was given a `Y/n` prompt when it reached this point. After entering `y`, the rest of the upgrade was successful, and `libc++1` was up-to-date after the upgrade.

So as long as we add `-y` to that `apt-get` command, that seems to be a working solution.
